src/bridge/cockpitfsread.c

```diff
@@ -238,6 +249,26 @@ cockpit_fsread_prepare (CockpitChannel *channel)
       return;
     }
 
+  if (fstat (self->fd, &statbuf) < 0)
+    {
+      cockpit_channel_fail (channel, "internal-error",
+                            "%s: couldn't stat: %s", self->path, strerror (errno));
```

Comment:
This doesn't need to be on a new line - exactly the same length as above https://github.com/cockpit-project/cockpit/pull/5767/files#diff-73773a0cf317a4ffe3ea7add9110f08cR222 where there is already only one line.